pkg/mount/state.go

```diff
@@ -31,6 +33,21 @@ type State struct {
 	fstabs       []*fstab.Mount
 }
 
+func (s *State) SortedBindMounts() []string {
+	bindMountsCopy := s.BindMounts
+	sort.Slice(bindMountsCopy, func(i, j int) bool {
+		iAry := strings.Split(bindMountsCopy[i], "/")
+		jAry := strings.Split(bindMountsCopy[j], "/")
+		iSize := len(iAry)
+		jSize := len(jAry)
+		if iSize == jSize {
+			return strings.Compare(iAry[len(iAry)-1], jAry[len(jAry)-1]) == -1
```

Comment:
goddamm, I miss python list comprehension/lambda . This would be a simple line in python :sob: